backend/app/celery/tasks/job_execution.py

```
@@ -232,6 +232,30 @@ def run_tts_batch_submission(
 
 
+@celery_app.task(
+    bind=True, queue="low_priority", priority=1, soft_time_limit=1800, time_limit=2100
+)
+def run_assessment_run(
+    self,
+    run_id: int,
+    organization_id: int,
+    project_id: int,
+    trace_id: str,
+    **kwargs,
```

Ruff (0.15.15)

[warning] Unused function argument: `kwargs`

(ARG001)